src/environments/environment.ts

```diff
@@ -2,8 +2,7 @@
 // `ng build --prod` replaces `environment.ts` with `environment.prod.ts`.
 // The list of file replacements can be found in `angular.json`.
 
-
-const serverUrl = 'https://146.48.93.234:9000/maia';
+const serverUrl = 'URL';
```

Comment:
@cienco l'idea è quella di rimuovere potenziali dati sensibili, quindi tieni te in locale l'indirizzo della macchina cui puntare senza poi inserirlo nei commit.

Reply:
perfetto si certo